META.coq

```diff
@@ -9,26 +9,26 @@
 #  linkopts(byte)   = "-dllpath ${coq_install_path}/kernel/byterun/ -dllib -lcoqrun"
 
 description = "The Coq Proof Assistant Plugin API"
-version = "8.7"
+version = "8.8"
```

Comment:
This version number change seems unrelated.

Reply:
I thought it was a good moment to bump as `META` indeed has non-backwards-compatible changes in this PR.